resources/views/widgets/settings/availability-map.blade.php

```diff
@@ -20,7 +20,7 @@
             <option value="1" @if($color_only_select == 1) selected @endif>{{ __('empty') }}</option>
             <option value="4" @if($color_only_select == 4) selected @endunless>{{ __('Display Name') }}</option>
             <option value="2" @if($color_only_select == 2) selected @endunless>{{ __('Hostname') }}</option>
-            <option value="3" @if($color_only_select == 3) selected @endunless>{{ __('Sysname') }}</option>
+            <option value="3" @if($color_only_select == 3) selected @endunless>{{ __('System Name') }}</option>
```

Comment:
if you change a name, please update all lang files

Reply:
I can’t find any entry for Sysname, so there should be no need for updating any key.

Reply:
a) I should be more specific: There is no trace of "Sysname" in any lang file (afaik), so a name change should be no problem.

b) Are that tests really stuck …? It is like this for a month now.